src/baselines.jl

```diff
@@ -91,18 +94,31 @@ end
 
 function selection_cut(wf::RDWaveform, base_uplim::Real, base_lolim::Real)
     # baseline start cut
-    base_start = wf.signal[1]
-    peak_index = risepoint(wf)
+    base_start = wf.signal[1] # value of waveform at the beginning
+    peak_index = risepoint(wf) # value of wf right before signal rising i.e. end of baseline
     baseline = wf.signal[begin:peak_index]
+    # is this supposed to remove pileup? i think it is base_slope() due to it not being flat
     cut_base::Bool = (base_start < base_uplim) && (base_start > base_lolim) && (base_start - mean(baseline) < 50)
 
+    # already return false if this one cut is false 
+    # otherwise there are bugs in base_slope()
+    if !cut_base
+        return false
+    end
+
     # wveform value cut 
     cut_value = wf.signal[1000] > 1000
 
     # peak cut 
     peak = findmax(wf.signal)[2]
     cut_peak = peak < 2100 && peak > 1650
```

Comment:
These seem like hard-coded values. Do they always apply or does it make sense to make these keyword arguments?

Reply:
Yeah this was done by a master student in our group years ago, and the values are hard-coded. Not sure if he tuned them for HADES data, or took them from quality cuts at the time. All this should be developed better later for L200.